src/meta/src/controller/streaming_job.rs

```diff
@@ -1214,12 +1214,22 @@ impl CatalogController {
         let inner = self.inner.read().await;
         let txn = inner.db.begin().await?;
 
+        {
+            let job = streaming_job::ActiveModel {
```

Comment:
Why updating `streaming_job` table here? IIUC we should update `rate_limit` for `source` table and notify source catalog back to FE.

Reply:
Looks like I missed out pushing some local changes. Fixed it.